### PR TITLE
attempt to fix existing functionality

### PR DESCRIPTION
**Pull request by idealllee**
The `patch-1` branch has the changes to add multiple thumbs and original PR here: https://github.com/jeanregisser/react-native-slider/pull/89/files

Though it breaks existing functionalities (see comments) below and this PR still doesn't fix the incorrect track highlighting issue.

### CODE REVIEW

File: src/Slider.js [hunk 1]
```diff
@@ -352,7 +353,7 @@ export default class Slider extends PureComponent {
       trackHighlightStyle = {
         position: 'absolute',
         left: 0,
-        width: Animated.add(thumbLeft, thumbSize.width / 2),
```
Comment (idealllee) [inline src/Slider.js]: `thumbLeft` was removed though still referenced here which gives us run time errors

File: src/Slider.js [hunk 2]
```diff
@@ -330,10 +330,11 @@ export default class Slider extends PureComponent {
 
     let trackHighlightStyle;
 
+    let flatValues = thumbValues.map(x => x.__getValue());
```
Comment (idealllee) [inline src/Slider.js]: attempt to fix a problem where the track highlight doesn't update with the single thumb movement. It updates fine with two thumbs. 
Another issue is that the tap-to-enter is broken with this version where slider thumb wouldn't move.

**Comment by drhops** [comment]
LGTM :trackball:

**Comment by idealllee** [comment]
close in favor of #2

**Pull request closed without merging**